docs/en/install-upgrade/upgrading-stack.asciidoc

```diff
@@ -65,15 +49,21 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 ===============================
 --
 
-. {ref}/docs-reindex.html[Reindex] or delete any indices created on 2.n. We recommend
-upgrading to the most recent 5.6 and using the {xpack} Reindex Helper to reindex 2.n indices.
-
-. If Kibana and {xpack} are part of your stack, upgrade the internal Kibana
-and {xpack} indices. We recommend using the {xpack} 5.6 Reindex Helper to
-upgrade the internal indices. If you're performing a full cluster restart upgrade
-from an earlier version, you can also <<upgrade-internal-indices,use the
- `_xpack/migration/upgrade` API>> directly to upgrade the
-internal indices after you install Elasticsearch {version}.
```

Comment:
This still applies to folks getting to 6.7.

Reply:
If you look at the preview, you'll see that's covered in the Upgrade Assistant step.  If it's still not clear, however, let me know.

Reply:
I removed the part about full cluster restarting from an earlier version, since this section assumed you were 5.6 or later, but will see if it's appropriate to add to lower sections.